sql/wsrep_sst.cc

```diff
@@ -1540,7 +1542,7 @@ static bool is_sst_request_valid(const std::string &msg) {
       Instead of this we will just allow alpha-num + a few special characters
       (colon, slash, dot, underscore, square brackets, hyphen). */
     std::string data = msg.substr(method_len + 1, data_len);
-    static const std::regex allowed_chars_regex("[\\w:/.[\\]-]+");
+    static const std::regex allowed_chars_regex("[\\w:/.[\\]@-]+");
```

Comment:
I think we don't need this change if we pass the address in the form `ip:port/path`

Reply:
I still think that having the path as you mention is not ok at least in standard cases.
If you check the Glaera documentation they say:
```
To signal that the node is ready to receive the state transfer, print the following string to standard output: ready <address>:port\n. Use the IP address and port at which the node is waiting for the state snapshot. For example:

ready 192.168.1.1:4444
```
So using the standard uri for db connection host:port\termination which in this case evolves to user:pw@host:port\n The use of uri/path was probably added for XB and other attributes

Reply:
As said in another comment, I inisist on using existing protocol.
The standard URI for the db connection is good thing, however, does not apply here.

Reply:
As said above, no.
standard is clear also in codership documentation they follow the `<address>:<ip>\n`
If in any part of the code we are deviationg from what is the real standard we should correct that.